_services_projects/forest-service.md

```diff
@@ -20,50 +20,25 @@ product_clients:
 resources:
     - "[e-Permit API task order](https://github.com/18F/bpa-fs-epermit-api)"
     - "[e-Permit intake task order](https://github.com/18F/bpa-fs-epermit-intake)"
-    - "[Christmas Tree task order](https://github.com/18F/bpa-fs-xmas-trees)"
+    - "[Christmas tree task order](https://github.com/18F/bpa-fs-xmas-trees)"
 ---
 
-The U.S. Department of Agriculture’s Forest Service issues permits to
-the public to conduct outfitting trips, large scale events, and cut down
-their very own Christmas tree on National Forest land.
+The U.S. Department of Agriculture’s Forest Service issues permits so the public can use National Forest lands for things like outfitting trips, organizing large scale events, and harvesting Christmas trees. 
```

Comment:
i think the previous framing is slightly more accurate as you are doing things on the land, rather than using the land to do those things.